addon/components/frost-select-dropdown.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     ]
       .join(';')
 
-    return Ember.String.htmlSafe(style)
+    return EmberString.htmlSafe(style)
```

Comment:
I agree with the first two, the second should never be needed, there shouldn't be another `Router` symbol in that file, it should be renamed. 

I believe the agreed upon approach here is to begin a discussion in `#frost-desgin` if we want to move away from the previous agreed-upon convention of descructuring everything *except* what clashes with a built-in and leaving those on the `Ember` object `Ember.String`, `Ember.Object` but *not* `Ember.Router`, in that case, the `var Router` should be changed (in case I didn't say that enough times yet ;)